webapp/components/Comment.vue

```diff
@@ -10,7 +10,7 @@
     </ds-card>
   </div>
   <div v-else :class="{ comment: true, 'disabled-content': comment.deleted || comment.disabled }">
-    <ds-card>
+    <ds-card :id="`commentId-${comment.id}`">
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-08-19T14:25:10Z" title="Monday, August 19th 2019, 4:25:10 pm +02:00">Aug 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -10,7 +10,7 @@
     </ds-card>
   </div>
   <div v-else :class="{ comment: true, 'disabled-content': comment.deleted || comment.disabled }">
-    <ds-card>
+    <ds-card :id="`commentId-${comment.id}`">
```


Do we need the prefix `commentId`?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-20T05:24:16Z" title="Tuesday, August 20th 2019, 7:24:16 am +02:00">Aug 20, 2019</time>_

---


@mattwr18 and I did it for the user experience. That a user would have a chance for a clue about what this is, even in case the scroll might not working. @alina-beck